### ISSUE TITLE
列表中显示多个gif，gif显示明显卡顿

### ISSUE THREAD

**Issue by chellChang** [SDWebImage/SDWebImage#3116]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by Insofan** [comment]
Could you provide a demo help us to reproduce it.

**Comment by chellChang** [comment]
hi:
Thank you for your reply. 

The demo is in the attachment.




      
                                        从QQ邮箱发来的超大附件              
                                                                     
                                       Gif_test.zip (6.31M, 无限期)                                           进入下载页面：http://mail.qq.com/cgi-bin/ftnExs_download?k=27336130b832aecff9c9cb6a1f35071f500351560d030d534b0658095d18010951034c095d05071d560258555c01065254025300393935770f553e445c46411e1c5a113004&t=exs_ftn_download&code=f3a09550&from=mobile&expiretime=-1&fsize=6.31M&suffix=zip             
         
     
 
                                          
                                       1604282788304105.mp4 (3.37M, 无限期)                                           进入下载页面：http://mail.qq.com/cgi-bin/ftnExs_download?k=2d3265340f14ff9efbc8cf6e4336074e05005c0c0405515249515207001b0159570248550154074c5d53555001020d525c57530765223550520251065d0402595c0155005406004f0942513458&t=exs_ftn_download&code=d2e4e65a&from=mobile&expiretime=-1&fsize=3.37M&suffix=mp4

**Comment by Insofan** [comment]
@chellChang I  update your project Sd version to `5.9.4` change FLAnimatedImage to UIImageView everything looks fine.
